.env

```diff
@@ -23,6 +23,7 @@ SYMFONY_DOCS_SECRET=''
 SYMFONY_SECRET=''
 SYMFONY_UX_SECRET=''
 SYMFONY_WEBPACK_ENCORE_SECRET=''
+SYMFONY_WEBPACK_ENCORE_BUNDLE_SECRET=''
```

Comment:
@fabpot I guess the secret needs to be set.
Can you please do it and merge this PR? Thanks